boms/cloud-oss-bom/pom.xml

```diff
@@ -52,8 +52,8 @@
     <protobuf.version>3.17.3</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
-    <gax.version>1.66.0</gax.version>
-    <gax.httpjson.version>0.83.0</gax.httpjson.version>
+    <gax.version>1.67.0</gax.version>
+    <gax.httpjson.version>0.84.0</gax.httpjson.version>
```

Comment:
This is not in Maven Central yet.

Reply:
Looks like the kokoro job failed:
http://sponge/4d59a9d5-e7af-429c-b234-da0188f6f7d9

I'm asking someone from yoshi to take a look